src/Exclusion/ExpressionLanguageExclusionStrategy.php

```diff
@@ -56,4 +56,26 @@ public function shouldSkipProperty(PropertyMetadata $property, Context $navigato
 
         return $this->expressionEvaluator->evaluate($property->excludeIf, $variables);
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function shouldSkipPropertyOnDeserialize(PropertyMetadata $property, Context $navigatorContext): bool
```

Comment:
I beilive that this method can be moved into `shouldSkipProperty` and use the `$navigatorContext` to understand if we are in "deserialization". In this way we do not need a new public method in this class.

Reply:
good idea, I've changed that